app/policies/form_policy.rb

```diff
@@ -72,6 +72,10 @@ def can_administer_group?
 
 private
 
+  def groups_enabled?
```

Comment:
Lovely. Clearly named method that tells you what it's doing before I have to read the code, and when I do read the code I know if it's actually doing what's expected based on the name. 👏  more please!